Update work and about me section

abdulrahmanshr75 opened the pull request:
- added the linters
- configured the site

abdulrahmanshr75 committed:
Install files

Installed all necessary files

abdulrahmanshr75 committed:
Update-My-Portfolio

abdulrahmanshr75 committed:
Update

Added the linters
configured the whole site html

keneogu commented on index.html:
```diff
@@ -0,0 +1,215 @@
+          </div>
+        </section>
+      </div>
+      <div class="second-container">
```
Let's make use of the `<section>` semantic tag instead of the `<div>` tag.

keneogu commented on index.html:
```diff
@@ -0,0 +1,215 @@
+                some of my work and experience! If you like what you see and
+                have a project you need coded, don’t hestiate to contact me.
+              </p>
+              <a class="button2" href="#">Get My Resume</a>
```
Add styling  for all your buttons with the `hover` `pressed` properties

keneogu commented on index.html:
```diff
@@ -0,0 +1,215 @@
+                  <li class="languagesdef">Javascript</li>
+                  <li class="languagesdef">html</li>
+                </ul>
+                <a class="button" href="#">See Project</a>
```
Add a styling for the buttons with `hover` `pressed` properties

keneogu reviewed (changes requested):
Hi @abdulrahmanshr75 , 

Good job so far!

- Linters are working
- well structured codes

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

abdulrahmanshr75 committed:
changes

- used <section> semantic tag instead of the <div>.
- Added styling for all your buttons with the hover pressed properties.

abdulrahmanshr75 committed:
Updated styles.css

rustam-tolipov reviewed (approved):
## Hi @abdulrahmanshr75  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

![image](https://i.pinimg.com/originals/77/0e/b0/770eb057449582c44672b2d2d1cc39b6.gif)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
